json/latest/brace-data-exchange.schema.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "$id": "https://brace-ai.github.io/schemas/json/v2.0.12/brace-data-exchange.schema.json",
+  "$id": "https://brace-ai.github.io/schemas/json/v2.0.14/brace-data-exchange.schema.json",
```

Comment:
will need to be 15 now

Reply:
Ah, another version slipped in before I could get this one merged 😅  Thank you! Fixed.